src/main/java/org/desarrolladorslp/technovation/repository/DeliverableRepository.java

```diff
@@ -22,4 +23,15 @@
             "SELECT batch_id FROM deliverables WHERE id = :deliverableId)"
             , nativeQuery = true)
     int assignDeliverableToSession(UUID deliverableId, UUID sessionId, String type);
+
+    @Modifying
+    @Query(value = "INSERT INTO deliverables_resources (deliverable_id, resource_id) VALUES (:deliverableId, :resourceId)", nativeQuery = true)
+    void addResourceToDeliverable(UUID deliverableId, UUID resourceId);
+
+    @Query("SELECT r FROM Resource r JOIN DeliverableResources dr ON dr.deliverableId = :deliverableId WHERE r.id = dr.resourceId")
+    List<Resource> getResourcesByDeliverable(UUID deliverableId);
```

Comment:
We need to validate that only MENTORS of the owner of the deliverable, ADMINISTRATORS and the owner of the deliverable can retrieve the resources of a specific deliverable. This can be added in a future issue, just create the issue please and add the link here.

Reply:
https://github.com/desarrolladorSLP/technovationslp-backend/issues/143